pkg/store/consul.go

```diff
@@ -81,8 +85,11 @@ func (s *Consul) setTimeout(time time.Duration) {
 }
 
 // SetEphemeralTTL sets the ttl for ephemeral nodes
-func (s *Consul) setEphemeralTTL(time time.Duration) {
-	s.ephemeralTTL = time
+func (s *Consul) setEphemeralTTL(ttl time.Duration) {
+	if ttl < MinimumTimeToLive {
+		log.Fatal("Consul does not allow a ttl < 10s, please specify a ttl >= 10s")
```

Comment:
I believe a library should never Fatal - we should return an error in initialize